### PR TITLE
[#16335] - Add space before "Forgot password?"

### LINKED ISSUE #16335: No space at "Wrong password" hint
# Bug Report

## Problem 
[comment]: # (A problem description)
IMAGE 2023-06-20 15:48:41

### Additional Information

- Status version: nightly 20/06/23
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by ulisesmac**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

fixes #16335

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)

Add a missing space before the "Forgot password?"" text

image

#### Platforms

- Android
- iOS

status: ready

### CODE REVIEW

File: src/status_im2/contexts/onboarding/profiles/view.cljs [hunk 1]
```diff
@@ -232,7 +232,6 @@
           error]
          [rn/touchable-opacity
           {:hit-slop       {:top 6 :bottom 20 :left 0 :right 0}
-           :style          {:margin-left -4}
```
Comment (ulisesmac) [inline src/status_im2/contexts/onboarding/profiles/view.cljs:235]: I removed this `:margin-left -4` to create the space between the words,
@ajayesivan it seems you added it, I removed it, but I was wondering why do we had it before, do you remember about it? :) thanks!

**Review by briansztamfater — APPROVED** [review]
LGTM!

**Comment by Parveshdhull** [comment]
Hi @ulisesmac, thank you very much for fixing it. Should we also fix the error message?
https://www.figma.com/file/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?type=design&node-id=2941-331293&t=TBNXNf3kqazgz3wr-0

image

**Comment by ulisesmac** [comment]
> Hi @ulisesmac, thank you very much for fixing it. Should we also fix the error message? https://www.figma.com/file/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?type=design&node-id=2941-331293&t=TBNXNf3kqazgz3wr-0
> 
> image

Yes, thanks @Parveshdhull 

Will fix it :+1:

File: src/status_im2/contexts/onboarding/profiles/view.cljs (hunk 1 above)
Comment (ajayesivan) [reply src/status_im2/contexts/onboarding/profiles/view.cljs:235]: I don't remember why I added it, but at that time it looked right.

**Review by ajayesivan — APPROVED** [review]
LGTM! Thanks

**Comment by VolodLytvynenko** [comment]
hi @ulisesmac thank you for PR. No issues from my side. Can be merged

File: src/status_im2/contexts/onboarding/profiles/view.cljs (hunk 1 above)
Comment (ulisesmac) [reply src/status_im2/contexts/onboarding/profiles/view.cljs:235]: Thank you so much!
 I was afraid I could be breaking something! :+1:

**Pull request merged**